examples/run_Hahn_echo.py

```diff
@@ -49,10 +56,26 @@ def verifyParams():
         print("VERIFIED DELAY TIME.")
     return
 #}}}
+#{{{get parameters from config file
 
+## JF 5/6 → 2 problems here:
+## (1) these need to be converted from a string to a number
```

Comment:
this does need to be fixed for this PR -- then delete relevant part of comment

Reply:
resolved- I merged in master for the right hahn echo ppg and figured how to convert from string to float for the values